components/mycart.css

```diff
@@ -0,0 +1,60 @@
+.cart-container {
+  margin: 5rem;
```

Comment:
if you are trying to center align, then give the container a width and use margin: auto instead of 5rem margin.